libs/@hashintel/type-editor/src/entity-type-editor/property-list-card/inherited-property-row.tsx

```diff
@@ -0,0 +1,29 @@
+import { usePropertyTypesOptions } from "../../shared/property-types-options-context";
```

Comment:
This file is mostly a straight extraction from `property-list-card`, with the exception of `InheritanceData` added as props, which affects the readonly error message shown